css/intro.css

```diff
@@ -77,6 +77,12 @@
 }
 
 @media screen and (max-width: 500px) {
```

Comment:
Why you have used **500px** as breakpoint here?

Reply:
To make 404 small in mobile and in ui form when i go below 500 the text get cut

can u check this two video 
### intro-page UI
https://user-images.githubusercontent.com/107163260/228812084-79506418-0856-4d70-beb6-1119046a795f.mp4

### Add 404 page(url access by super user)
https://user-images.githubusercontent.com/107163260/228411773-e7d41b5d-4171-4514-854f-0f625befdfb1.mp4


To